src/System.Numerics.Vectors/tests/GenericVectorTests.cs

```diff
@@ -1909,7 +1910,7 @@ static GenericVectorTests()
                 (index, val) =>
                 {
                     T expected = Util.Sqrt(values[index]);
-                    Assert.Equal(expected, val);
+                    AssertEqual(expected, val, $"SquareRoot( {values[index]} )", precision);
```

Comment:
Should we also update the operation string to output 9 digits and 17 digits for float/double? Otherwise, we won't know the actual input value that is causing the failure.